### PR TITLE
CatalogOfMyThings

### PR DESCRIPTION
**Pull request by SimonGrchevski**
In this project, me and my partners, @olawale-o, @elasfarc finished: 
- Create Item class
- Create a Book class
- Create a Label class with an association to the Item class
- Create MusicAlbum class
- Create Genre class with an association to the Item class
- Create a Game class
- Create an Author class with an association to the Item class
- All Item class properties visible in the diagram are defined and set up in the constructor method
- Implement  all methods for the classes
- Create a main.rb file that will serve as your console app entry-point
- Implement startup actions
- Add unit tests for all implemented methods in each class
- All data is preserved by saving collections in .json files
- Add a schema.sql file with tables that are  analogical to the structure of the classes
- Follow SOLID, DRY, and KISS approaches

[Video](https://drive.google.com/file/d/1zR-ZyVHcJpOcMIjRIv16HautwyW3d7X9/view?usp=sharing)

### CODE REVIEW

File: creator.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  def create_game
+    print 'Is it Multiplayer? [Y/N]: '
+    multi_player = user_input == 'n'
+    print 'Last Played At (yyyy-mm-dd): '
+    last_played_at = user_input
+    game = Game.new(!multi_player, last_played_at, { publish_date: publish_date, archived: !archived })
+    @games << game
+    create_associations(game)
+    puts 'Game created successfully'
+  end
```
Comment (Emmyn5600) [inline creator.rb]: - [ ] Let us throw an error while a user enters a wrong format date `2021-20-20`, you need to format the user_input mostly on published_date or elsewhere related to the date. Your format is (yyyy-mm-dd) so try to validate it.

Make sure the validation of format date is working in all features that have date format.

Hint:  ` 'publish_date' => @publish_date.strftime('%Y/%m/%d'), `

date

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi,

Good job so far! 👏🏼 @elasfarc , @olawale-o   and @SimonGrchevski 

May I highlight the following:
- 9 minutes of video presentation is showing all the features of the project:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- The git-flow has been followed :heavy_check_mark:
- The repo looks professional:heavy_check_mark:
- Your project looks awesome:heavy_check_mark: 
- The database is well structured :heavy_check_mark: 
- The data are being stored in JSON files :heavy_check_mark: 
- The project is running successfully:heavy_check_mark: 
- 38 Tests are passing successfully :heavy_check_mark: 

## STATUS: CHANGES REQUESTED :recycle:


### Changes requested.



- _Setting up the linters `linters.yml file` needs to be inside the `.github/workflows`_
- _Check the comment under the review_

 #### Optional suggestions
 
 - N/A
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: creator.rb (hunk 1 above)
Comment (olawale-o) [reply creator.rb]: Hi @Emmyn5600
Thank you for pointing out these

Comment (Emmyn5600) [reply creator.rb]: Well solved

Comment (olawale-o) [reply creator.rb]: Hi @Emmyn5600
Thank you for the follow-up. We really appreciate it

**Review by Emmyn5600 — APPROVED** [review]
Hi,Team

Good job so far! 👏🏼

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer.

 #### Optional suggestions
 
 - N/A
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by olawale-o** [comment]
Hi @Emmyn5600
Thank you for your time and effort on the review

**Pull request merged**